kotlin-eclipse-ui/src/org/jetbrains/kotlin/ui/editors/quickassist/KotlinQuickAssistProposal.java

```diff
@@ -40,15 +42,27 @@ public IFile getActiveFile() {
     }
     
     public int getStartOffset(@NotNull PsiElement element, @NotNull AbstractTextEditor editor) {
```

Comment:
I believe that in this method and in method `getEndOffset` we always can retrieve IFile because these methods are calling after several similar checks. So the assertion for notnull will be enough